submissions/michaelolubajo/introductionToHtml/productpage.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Mi Band 5 Product Page</title>
+</head>
+<body>
+	<a href=""><img src="images/logo.png"></a>
+	<div class="topnav">
+  		<input type="text" placeholder="Search..">
+	</div>
+	<div class="topnav">
+  		<a class="active" href="#home">Home</a>
+  		<a href="#phones">Phones</a>
+ 		<a href="#mobilephonesaccessories">Mobile Phone Accessories</a>
+  		<a href="#smartwatches">Smart Watches</a>
+  		<a href="#miband5">Mi Band 5</a>
+	</div>
+	<hr>
+	<h2>Mi Band 5</h1>
+	<img src="images/1-mi-band-5.jpg" height="400" width="400">
```

Comment:
Remove the height and width in the images

Reply:
Done.

Reply:
![Screenshot_2020-07-20 Mi Band 5 Product Page](https://user-images.githubusercontent.com/46163342/87924004-31e0f700-ca76-11ea-879a-4899e753cda4.png)